CONTRIBUTING.md

```diff
@@ -20,18 +18,17 @@ Anyone can contribute to USWDS. Whether it's submitting a bug or proposing a new
 
 First time contributor? We’re here to help guide you through a successful contribution. We review all contributions before merging them into USWDS. If you’re unsure about anything, just [ask](mailto:uswds@gsa.gov) — or submit your issue or pull request anyway to get the conversation started.
 
-Before submitting a contribution, you’ll just need to create a GitHub account or sign in to your existing account.  
-
-If you want to see some other contributions before submitting your own, check out [some good first issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Good+First+Issue%22) from other community members. Again, if you’ve got questions, don’t hesitate to [reach out to us](mailto:uswds@gsa.gov). 
+Before submitting a contribution, you’ll just need to create a GitHub account or sign in to your existing account.
 
+If you want to see some other contributions before submitting your own, check out [some good first issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Good+First+Issue%22) from other community members. Again, if you’ve got questions, don’t hesitate to [reach out to us](mailto:uswds@gsa.gov).
 
-###  Submitting bugs and issues
+### Reporting bugs
```

Comment:
**suggestion:** Recommend keeping issues for non-technical users.

```suggestion
### Reporting bugs and issues
```

Reply:
Updated in [f12bc9a](https://github.com/uswds/uswds/pull/6038/commits/f12bc9af6ada0ef898748464e340ba77fe9a302a)